sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -100,7 +100,13 @@ def add_dependency_management_for_file(file_path, spring_boot_dependencies_versi
     log.info("Add dependency management for file: " + file_path)
     with open(file_path, 'r', encoding = 'utf-8') as pom_file:
         pom_file_content = pom_file.read()
+        if pom_file_content.find("<dependencyManagement>") != -1:
```

Comment:
I think [this change](https://github.com/Azure/azure-sdk-for-java/blob/3c8d789cae9c008604e6b276cec858aae71a6958/sdk/spring/scripts/compatibility_insert_dependencymanagement.py#L104-L107) is better, because we may also want to test the compatibility with test modules. And I think the log info in that commit is also more informative.